src/services/repositories/external/ConnectorJobRepository.php

```diff
@@ -68,6 +70,62 @@ public function start(int $liltJobId): bool
      */
     public function findOneById(int $liltJobId): JobResponse
     {
-        return $this->apiInstance->servicesApiJobsGetJobById($liltJobId);
+        $cacheKey = __METHOD__ . ':' . $liltJobId;
+        $cache = CraftliltpluginParameters::getResponseCache();
+
+        if (!empty($cache)) {
```

Comment:
There is already a check for this in `getResponseCache()`. Do we need another one here?